record/record_test.go

```diff
@@ -332,22 +332,52 @@ func writeRecords(records [][]byte) ([]byte, error) {
 	return buf.Bytes(), nil
 }
 
-func TestBasicRecover(t *testing.T) {
-	records := [][]byte{
-		[]byte(strings.Repeat("a", blockSize-headerSize)),
-		[]byte(strings.Repeat("b", blockSize-headerSize)),
-		[]byte(strings.Repeat("c", blockSize-headerSize)),
+// generateTestRecords returns up to 256 records in a slice of slices of the specified lengths.
```

Comment:
Why up to 256? If i is the int 258, then byte(i) is perfectly well defined in Go: it is equal to 2.